src/namespaces.ts

```diff
@@ -0,0 +1,15 @@
+import namespace from '@rdfjs/namespace';
+
+const namespaces = {
```

Comment:
`namespaces` is constant, but any way to make the values of the object constant as well? They can be reassigned and it is an exported variable :thinking: